### PR TITLE
Added service user warning and removed PCI content

### PR DESCRIPTION
**Pull request by bryanstephenson**

### CODE REVIEW

File: xml/security-security_features.xml [hunk 1]
```diff
@@ -6,7 +6,32 @@
  <!ENTITY % entities SYSTEM "entities.ent"> %entities;
 ]>
 <chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="newsecurity">
- <title>&kw-hos-tm;: Security Features Overview</title>
+ <title>&kw-hos-tm;: Security Planning and Features</title>
+ <section xml:id="idg-all-security-security_planning-xml-1">
+  <title>Security planning</title>
+  <para>
+   &kw-hos-phrase; is a complex system which has many options which has many options which
```
Comment (guangyee) [inline xml/security-security_features.xml]: s/which has many options which has many options/which has many options/

Comment (bryanstephenson) [reply xml/security-security_features.xml]: Thanks for the catch.

File: xml/security-security_features.xml [hunk 2]
```diff
@@ -6,7 +6,32 @@
...
+   environment. You should read
+   <link xlink:href="https://docs.openstack.org/security-guide/introduction/security-boundaries-and-threats.html"/>.  
+   to understand how OpenStack defines security domains. Ensure that the network layout
+   you choose for your deployment provides adequate network separation for your specific
+   requirements. Take advantage of the security features described below in this section. 
+  </para>
+  <para>
+   OpenStack service processes need to run some commands as root in order to provide
+   functionality. The service processes run as non-root users but can escalate privilege
+   to root to perform certain operations. If a service process is compromised to the
```
Comment (guangyee) [inline xml/security-security_features.xml]: Maybe we need to document how oslo.privsep works and the default filters for each service?

https://docs.openstack.org/oslo.privsep/latest/

Per my understanding, we have some control over the type of commands just like sudo.

File: xml/security-security_features.xml [hunk 3]
```diff
@@ -6,7 +6,32 @@
...
+   extent that an attacker can execute arbitrary commands as the service user then it is
+   likely that the attacker can elevate privilege to root and take over the system on
+   which the service is running. When planning the security controls for an OpenStack
+   deployment you should consider the services to be running as root when deciding which
+   security controls are appropriate for your environment. Timely installation of
+   security patches is essential. Other controls such as Intrusion Prevention Systems
+   and Web Application Firewalls should be considered.
```
Comment (guangyee) [inline xml/security-security_features.xml]: This paragraph read like it's tailored for lawyers. That the intention? For operation folks, maybe we need to point them to https://docs.openstack.org/security-guide/?

File: xml/security-security_features.xml (hunk 2 above)
Comment (bryanstephenson) [reply xml/security-security_features.xml]: I think that's more than we want to get into at this time.

File: xml/security-security_features.xml (hunk 3 above)
Comment (bryanstephenson) [reply xml/security-security_features.xml]: I will add a pointer to https://docs.openstack.org/security-guide/

File: xml/security-security_features.xml [hunk 4]
```diff
@@ -6,7 +6,34 @@
  <!ENTITY % entities SYSTEM "entities.ent"> %entities;
 ]>
 <chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="newsecurity">
- <title>&kw-hos-tm;: Security Features Overview</title>
+ <title>&kw-hos-tm;: Security Planning and Features</title>
+ <section xml:id="idg-all-security-security_planning-xml-1">
+  <title>Security planning</title>
+  <para>
+   &kw-hos-phrase; is a complex system which has many options which
```
Comment (cmurphy) [inline xml/security-security_features.xml]: Is this a HOS-specific document?

File: xml/security-security_features.xml [hunk 5]
```diff
@@ -6,7 +6,34 @@
...
+   affect the security of the system and the workloads running within it. Careful planning
+   is required to ensure the correct options and features are chosen for your particular
+   environment. You should read
+   <link xlink:href="https://docs.openstack.org/security-guide/introduction/security-boundaries-and-threats.html"/>.
```
Comment (cmurphy) [inline xml/security-security_features.xml]: Don't include a raw link, use a title in the link, such as `"Read the <link xlink:href="...">Security Boundaries and Threats Guide</link>"`

File: xml/security-security_features.xml [hunk 6]
```diff
@@ -6,7 +6,34 @@
...
+  <para>
+   &kw-hos-phrase; is a complex system which has many options which
+   affect the security of the system and the workloads running within it. Careful planning
+   is required to ensure the correct options and features are chosen for your particular
+   environment. You should read
+   <link xlink:href="https://docs.openstack.org/security-guide/introduction/security-boundaries-and-threats.html"/>.  
+   to understand how OpenStack defines security domains. Ensure that the network layout
+   you choose for your deployment provides adequate network separation for your specific
+   requirements. The other parts of the OpenStack Security Guide at
+   <link xlink:href="https://docs.openstack.org/security-guide/"/> are also informative.
```
Comment (cmurphy) [inline xml/security-security_features.xml]: Same here

File: xml/security-security_features.xml (hunk 4 above)
Comment (bryanstephenson) [reply xml/security-security_features.xml]: No, but this keyword expands to "SUSE OpenStack Cloud 9" so it's OK. The "hos" in the name of the keyword was never removed.

**Comment by bryanstephenson** [comment]
Other changes have happened in this file. Rather than do a rebase I will abandon this PR and perform the changes in a new PR.

**Pull request closed without merging**